docs/src/operations.md

```diff
@@ -211,3 +211,110 @@ BinaryOperation at (Center, Center, Center)
         │   └── 4×1×4 Field{Center, Center, Center} on RectilinearGrid on CPU
         └── 2
 ```
+
+## Averages and integrals
+
+Let's demonstrate now how we can compute the average or the integral of a field over the grid or over some part of the grid.
+
+We start by creating a latitude-longitude grid that only goes up to 30 degrees latitude.
+Conveniently, with this latitude extend that grid covers half the total area of the sphere, i.e., `2π * grid.radius^2`.
+
+Let's try to estimate this are using `Integral` operation. We create a Field, we fill it with ones and we integrate it over the whole grid.
+
+
+```jldoctest operations_avg_int
+using Oceananigans
+
+grid = LatitudeLongitudeGrid(CPU(); size=(60, 10, 5),
+                                    latitude = (-30, 30),
+                                    longitude = (0, 360),
+                                    halo=(7, 7, 7),
+                                    z=(-1000, 0))
+
+c = CenterField(grid)
+
+set!(c, 1)
+
+∫c = Field(Integral(c, dims=(1, 2)))
+
+# output
+1×1×5 Field{Nothing, Nothing, Center} reduced over dims = (1, 2) on LatitudeLongitudeGrid on CPU
+├── data: OffsetArrays.OffsetArray{Float64, 3, Array{Float64, 3}}, size: (1, 1, 5)
+├── grid: 60×10×5 LatitudeLongitudeGrid{Float64, Periodic, Bounded, Bounded} on CPU with 7×7×7 halo and with precomputed metrics
+├── operand: Integral of BinaryOperation at (Center, Center, Center) over dims (1, 2)
+├── status: time=0.0
+└── data: 1×1×19 OffsetArray(::Array{Float64, 3}, 1:1, 1:1, -6:12) with eltype Float64 with indices 1:1×1:1×-6:12
+    └── max=0.0, min=0.0, mean=0.0
+```
+
+Few remarks. Note that the `∫c` has locations `Nothing, Nothing, Center`; this is because we have integrated in the first two dimensions. Further notice that there are no values in `∫c`, i.e.,
```

Comment:
```suggestion
Few remarks. Note that the `∫c` has locations `Nothing, Nothing, Center`; this is because we have integrated in the first two dimensions.
Further note that `∫c` is full of zeros,
```

Reply:
I don't think you have to show the values. You can already see above that the max, min, and mean are all 0.

Reply:
Fair point! `interior` is demonstrated anyway in the Field section